python/lsst/analysis/drp/dataSelectors.py

```diff
@@ -258,12 +258,52 @@ def __call__(self, df, **kwargs):
             An array with the objects that are classified as
             unknown marked with a 9.
         """
-        unknowns = (df[self.band + "_extendedness"] == 9.0)
+        unknowns = (df[self.band + "extendedness"] == 9.0)
         sourceType = np.zeros(len(df))
         sourceType[unknowns] = 9
         return sourceType
 
 
+class VisitPlotFlagSelector(DataFrameAction):
+
+    @property
+    def columns(self):
+        flagCols = ["psfFlux_flag", "pixelFlags_saturatedCenter", "extendedness_flag", "centroid_flag"]
+        yield from flagCols
+
+    def __call__(self, df, **kwargs):
+        """The flags to use for selecting sources for visit QA
+
+        Parameters
+        ----------
+        df : `pandas.core.frame.DataFrame`
+
+        Returns
+        -------
+        result : `numpy.ndarray`
+            A mask of the objects that satisfy the given
+            flag cuts.
+
+        Notes
+        -----
+        These flags are taken from pipe_analysis and are considered to
+        be the standard flags for general QA plots. Some of the plots
+        will require a different set of flags, or additional ones on
+        top of the ones specified here. These should be specifed in
+        an additional selector rather than adding to this one.
+        """
+
+        result = None
+        flagCols = ["psfFlux_flag", "pixelFlags_saturatedCenter", "extendedness_flag", "centroid_flag"]
```

Comment:
There is a selector that takes flag names as a config but it was decided to have selectors that stayed the same so that people could be sure they were reusing the same things when making plots.

Reply:
Hmm. I'm looking at line 271 in this same class, though, under the property `def columns(self)` you use repeatedly elsewhere, and not understanding why you can't use that here instead of redefining `flagCols` from scratch to match — it's the same four flags. Is this a special case for some reason?